sql/sql_explain.h

```diff
@@ -533,6 +537,8 @@ class Explain_query : public Sql_alloc
 #ifndef DBUG_OFF
   bool can_print_json= false;
 #endif
+
+  Exec_time_tracker query_optimization_total_time_tracker;
```

Comment:
This is a very long name. Please change for a shorter one, `optimization_time_tracker`.